modules/ocf_jenkins/manifests/init.pp

```diff
@@ -17,6 +17,14 @@
     stage => first,
   }
 
+  file {
```

Comment:
logically shouldn't this be `/etc/ocf-kubernetes/secrets` since it's reasonable to expect there may be other k8s-related things we may want on machines of this class in the future

Reply:
whoops that's a typo thanks